addon/controllers/dashboard/week.js

```diff
@@ -0,0 +1,3 @@
+import Controller from '@ember/controller';
+
+export default class DashboardWeekController extends Controller {}
```

Comment:
This is the same as not having the file at all, so it can probably be removed.

Reply:
:heavy_check_mark: